inc/agent.class.php

```diff
@@ -588,22 +588,27 @@ public function pre_deleteItem() {
             'id'  => $this->fields['users_id'],
       ], true);
 
+      // Send unrolling message to reset agent status
+      if ($this->fields['enroll_status'] == 'enrolled') {
+         $this->sendUnenrollQuery();
+      }
+
       // Delete the MQTT user for the agent
-      if (!empty($serial)) {
+      /*if (!empty($serial)) {
```

Comment:
I think we shall delete the MQTT account because the agent is deleted.

Reply:
ok

Reply:
When we unenroll a device the deletion of the agent occurs when the ACK is received.

see https://github.com/flyve-mdm/flyve-mdm-glpi-plugin/blob/develop/inc/mqtthandler.class.php#L124